.github/workflows/inspektor-gadget.yml

```
@@ -454,6 +466,68 @@ jobs:
 
         perl tools/check-readme.pl ./kubectl-gadget README.md
 
+  test-integration-aks:
+    name: Integration tests on AKS
+    # level: 2
+    needs: [check-secrets, test-unit, build-kubectl-gadget, build-local-gadget, build-gadget-container-images]
+    #needs: [check-secrets, test-unit, build-kubectl-gadget, build-local-gadget, build-gadget-container-images, publish-gadget-images-manifest]
+    if: needs.check-secrets.outputs.aks == 'true'
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    permissions:
      contents: read
      # Required to be able to push the integration test image
      packages: write
```

Now that it will also run when making a release, I think it is needed. cc @mqasimsarfraz @mauriciovasquezbernal

---

Could you please rebase this PR now that https://github.com/kinvolk/inspektor-gadget/pull/955 has been integrated?